frontend/src/components/user/UserAvatar.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <v-avatar :size="size" :color="color">
+    <span class="white--text" :style="style">{{ initials }}</span>
+  </v-avatar>
+</template>
+<script>
+
+export default {
+  name: 'UserAvatar',
+  props: {
+    size: { type: Number, required: false, default: 48 },
+    value: { type: Object, required: true }
+  },
+  computed: {
+    user () {
+      if (this.value.user instanceof Function) {
+        return this.value.user()
+      }
+      if (this.value.user instanceof Object) {
+        return this.value.user
+      }
+      return this.value
+    },
+    color () {
+      const id = this.user.id
+      const h = (parseInt(id, 16) % 360)
+      return `hsl(${h}, 100%, 40%)`
+    },
+    initials () {
```

Comment:
Die Funktion initials() führt zu mehrere Konsolen-Fehlern beim Neuladen. Ich glaube, das kommt daher wenn this.user noch am Laden ist. 

Loading Skeleton, welches auf `this.user._meta.loading` hört?

Reply:
sollte gelöst sein.